ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -174,6 +172,14 @@
       </B.Tr>
     </:body>
   </Hds::Table>
+  {{#if @model.isLoadIncomplete}}
+    <Hds::Alert @type='compact' as |A|>
```

Comment:
do we not want to show it as a warning type in the updated design?

Reply:
Yeah I think we're changing it from a warning now to just be informational